packages/split-button/README.md

```diff
@@ -1,11 +1,11 @@
 ## Description
 
-`sp-splitbutton` delivers a
+An `sp-split-button` surfaces an immediately envokable action via it's main button, as well as a list of alternative actions in its toggleable menu overlay. By default, any actions envoked from the overlay will replace the main action button. When leveraging `[type="more"]` the action will be envoked, but the main button will remain the action initally persribed by the implementor.
```

Comment:
I'd suggest changing `envoke` to `invoke` since it's more common (and I know my spell check is really wanting to change it as I'm trying to give this feedback). I did learn today that `envoke` is a valid alternate spelling though 🌈